homeassistant/components/advantage_air/climate.py

```diff
@@ -72,35 +73,70 @@ async def async_setup_entry(
     entities: list[ClimateEntity] = []
     if aircons := instance["coordinator"].data.get("aircons"):
         for ac_key, ac_device in aircons.items():
-            entities.append(AdvantageAirAC(instance, ac_key))
+            entities.append(AdvantageAirAC(instance, ac_key, config_entry))
             for zone_key, zone in ac_device["zones"].items():
                 # Only add zone climate control when zone is in temperature control
-                if zone["type"] != 0:
+                if zone["type"] > 0:
                     entities.append(AdvantageAirZone(instance, ac_key, zone_key))
     async_add_entities(entities)
 
 
 class AdvantageAirAC(AdvantageAirAcEntity, ClimateEntity):
     """AdvantageAir AC unit."""
 
+    _attr_fan_modes = [FAN_LOW, FAN_MEDIUM, FAN_HIGH]
     _attr_temperature_unit = UnitOfTemperature.CELSIUS
     _attr_target_temperature_step = PRECISION_WHOLE
     _attr_max_temp = 32
     _attr_min_temp = 16
-    _attr_fan_modes = [FAN_AUTO, FAN_LOW, FAN_MEDIUM, FAN_HIGH]
-    _attr_hvac_modes = AC_HVAC_MODES
-    _attr_supported_features = (
-        ClimateEntityFeature.TARGET_TEMPERATURE | ClimateEntityFeature.FAN_MODE
-    )
+    _attr_preset_modes = [ADVANTAGE_AIR_MYZONE]
```

Comment:
Looks like this need to include the other modes

Reply:
Removing since this PR doesnt support any presets.

Reply:
Oh but you aren't implementing `set_preset_mode`  (maybe by design?)
https://developers.home-assistant.io/docs/core/entity/climate?_highlight=climate#set-preset-mode

Reply:
Yes, this PR is intentionally removing the ability to change presets since thats causing all the issues on #75246